HHS/Head-Start-TTADP
# [TTAHUB-3013] Do not roll up goal by status on reports

## Description of change
There is an ongoing bug where goals are separated by status on an Activity report. We've made many changes to how goal status is managed in the past months. For one, status is no longer editable on a report. We've also removed draft, closed, and suspended goals as options, and goals are moved from not started to in progress automatically. I don't think we need to separate them anymore.

Doing so also resolves this support case, where it is impossible to remove a phantom objective from a report.

## How to test
Use the new bin/latest_backup.sh script to download a copy of prod. 
Impersonate the user in the video linked in the Jira ticket.
Remove the phantom objective from her report and confirm that it stays gone.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-3013


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
